cns/restserver/api_test.go

```diff
@@ -1292,6 +1293,34 @@ func TestCreateHostNCApipaEndpoint(t *testing.T) {
 	fmt.Printf("createHostNCApipaEndpoint Responded with %+v\n", createHostNCApipaEndpointResponse)
 }
 
+func TestNCIDCaseInSensitive(t *testing.T) {
+	setEnv(t)
+	err := setOrchestratorType(t, cns.Kubernetes)
+	if err != nil {
+		t.Fatalf("TestNCIDCaseSensitive failed with error:%+v", err)
+	}
+
+	// add a list of NCIDs with upper-case NCIDs, lower-case NCIDs, upper-case cns-managed mode NCID and lower-case cns-managed mode NCID
+	ncids := []string{
+		strings.ToUpper("Swift_" + uuid.New().String()), strings.ToUpper("Swift_" + uuid.New().String()), "Swift_" + uuid.New().String(),
```

Comment:
Your earlier implementation was clearer. It showed the list of all possible NCIDs. I would suggest not using uuid but for better clarity and readability, use distinct guids.